source/guide/cordova-plugins.md

```diff
@@ -147,3 +147,61 @@ export default {
 }
 </script>
 ```
+
+### Example: Device
+First step is to read the documentation of the Cordova plugin that we want to use. We look at [Cordova Plugins list](https://cordova.apache.org/docs/en/latest/#plugin-apis) and click on [Device doc page](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-device/index.html).
+
+This plugin initializes a global variable called `device` which describes the device's hardware and software. As Quasar is the one who captures this event, you need modify the .eslintrc.js file, leaving the property globals in this way:
+
+```
+globals: {
+    'ga': true, // Google Analytics
```

Comment:
I am not sure we should put the **ga** here... Do we reference this anywhere else, i.e. where to put the ga id?